lib/lists/indexing/indexing_item_list.dart

```diff
@@ -53,7 +53,7 @@ class IndexingItemList extends ConsumerWidget {
                                     if (_item[f.data()['value']] != null) {
                                       if (type == 'Array of objects') {
                                         final objects = _item[f.data()['value']];
-                                        return objects.map((obj) => obj[nameFieldValue.text].toString()).join(", "); // 
+                                        return objects.map((obj) => obj[nameFieldValue.text] != null ? obj[nameFieldValue.text].toString() : obj[nameFieldValue.text]=' ').join(", ");
```

Comment:
let's return null value instead of ' ' and then remove nulls before joining. This is to avoid seeing: ', , , ,'. Instead we will see ''